ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml

```diff
@@ -126,6 +137,26 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-azure
   timeout: 5h0m0s
+- always_run: false
+  as: control-plane-multi-ipsec-3nodes
```

Comment:
the name here is a bit confusing since we get `"workerNodesCount":4,` from the json output.

Think lets update this pr to have the proper number of worker nodes for our 4.17 regression testing. But might be blocked by the issue Qiujie talked about today

Reply:
On it!